teaser/data/output/modelicatemplate/AixLib/AixLib_ThermalZoneRecord

```diff
@@ -0,0 +1,281 @@
+within ${mod_prj}.${bldg.name}.${bldg.name}_DataBase;
+record ${bldg.name}_${zone.name.replace(" ", "")} "${bldg.name}_${zone.name.replace(" ", "")}"
+  extends AixLib.DataBase.ThermalZones.ZoneBaseRecord(
+    T_start = ${zone.t_inside},
+    VAir = ${zone.volume},
+    AZone = ${zone.area},
+    alphaRad = ${mean_alphaRad(zone)},
+    lat = ${deg_to_rad(check_lat(bldg.latitude))},
+    nOrientations = ${def_nOrientations(add_gf(len(zone.tilt_wall),zone.area_gf,number_of_elements))},
+    AWin = {${list_to_string(add_gf_list(zone.window_area_list,zone.area_gf,number_of_elements))}},
+    ATransparent = {${list_to_string(add_gf_list(zone.window_area_list,zone.area_gf,number_of_elements))}},
+    alphaWin = ${zone.alpha_conv_inner_win},
+    RWin = ${def_val(zone.r1_win-R_zero(zone.alpha_comb_outer_win,zone.area_win))},
+    gWin = ${zone.weighted_g_value},
+    UWin= ${zone.ua_value_win/zone.area_win},
+    ratioWinConRad = ${zone.windows[0].a_conv},
+    AExt = {${list_to_string(add_gf_area(zone.outer_walls_areas,zone.area_gf,number_of_elements))}},
+    alphaExt = ${zone.alpha_conv_inner_ow},
+    nExt = 1,
+    RExt = {${def_val(zone.r1_ow)}},
+    RExtRem = ${def_val(zone.r_rest_ow-R_zero(zone.alpha_comb_outer_ow,zone.area_ow))},
+    CExt = {${def_val(zone.c1_ow)}},
+% if number_of_elements < 2:
+    AInt = 0,
+% else:
+    AInt = ${zone.area_iw},
+% endif
+    alphaInt = ${zone.alpha_conv_inner_iw},
+    nInt = 1,
+    RInt = {${def_val(zone.r1_iw)}},
+    CInt = {${def_val(zone.c1_iw)}},
+% if number_of_elements < 3:
+    AFloor = 0,
+% else:
+    AFloor = ${zone.area_gf},
+% endif
+    alphaFloor = ${zone.alpha_conv_inner_gf},
+    nFloor = 1,
+    RFloor = {${def_val(zone.r1_gf)}},
+    RFloorRem =  ${def_val(zone.r_rest_gf-R_zero(zone.alpha_comb_outer_gf,zone.area_gf))},
```

Comment:
I think the calculation of resistances should be implemented into thermal zone, instead of in the templates! (also for unit test reasons!)

AixLib + Annex various occurences